setup

```diff
@@ -230,11 +267,11 @@ def main():
 
     if arguments.action == "all":
         run_modules_storage_setup_action()
-        run_build_action()
+        run_build_action(arguments)
```

Comment:
I can't mark it above, but we should change `arguments` to `args`, so it is consistent everywhere.